code42.json

```diff
@@ -1924,6 +1924,463 @@
                 "type": "table"
             },
             "versions": "EQ(*)"
+        },
+        {
+            "action": "create case",
+            "identifier": "create_case",
+            "description": "Create a Code42 case",
+            "type": "investigate",
```

Comment:
Thinking about it a bit, I almost think all of our actions should be in the 'investigate' bucket, since (I think) 'contain' is much more of a response action to a particular event (lock down a device if ransomware detected, etc.), and most Code42 activity fits more into the investigate/observe category.